stream-chat-android-compose/GUIDELINES.md

```diff
@@ -5,27 +5,49 @@
 * [Motivation](#motivation)
 * [Project Structure](#project-structure)
 * [Writing documentation](#writing-documentation)
-  * [KDoc Comments](#kdoc-comments)
+    * [KDoc Comments](#kdoc-comments)
 * [Naming Conventions](#naming-conventions)
-  * [Naming Components](#naming-components)
-  * [Naming Listeners](#naming-listeners)
-  * [Naming State](#naming-state)
-  * [Naming Resources](#naming-resources)
+    * [Naming Components](#naming-components)
+    * [Naming Listeners](#naming-listeners)
+    * [Naming State](#naming-state)
+    * [Naming Resources](#naming-resources)
 * [Customizing Components](#customizing-components)
-  * [Customization Options](#customization-options)
-  * [Slot APIs](#slot-apis)
+    * [Customization Options](#customization-options)
+    * [Slot APIs](#slot-apis)
 * [Drawable colors](#drawable-colors)
-* [Component Previews](#component-previews)  
+* [Component Previews](#component-previews)
+* [Performance Guidelines](#performance-guidelines)
+    * [Write Stable Classes](#write-stable-classes)
+    * [Real World Implications](#real-world-implications)
```

Comment:
We published a new article following our performance guidelines.
Can we also mention this post somewhere? [6 Jetpack Compose Guidelines to Optimize Your App Performance](https://getstream.io/blog/jetpack-compose-guidelines/)

Reply:
We'll work on condensing this info and publishing a shorter set of guidelines, by pointing to the post first, for people to learn the basics.